app/controllers/application_controller.rb

```diff
@@ -1,5 +1,8 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    @current_user ||= User.first
+  before_action :configure_devise_parameters, if: :devise_controller?
+  def configure_devise_parameters
+    devise_parameter_sanitizer.permit(:sign_up) do |user|
+      user.permit(:name, :email, :password, :password_confirmation)
+    end
   end
 end
```

Comment:
- [ ] **[OPTIONAL]** I think it would be a good idea to restrict the app to only be accessible to users who are signed in. You can do that by adding the code below to this application controller. This way when a user is not logged in and tries to visit any other routes they'll be redirected to the sign in page.

  ```
  before_action :authenticate_user!
  ```

Reply:
This is a good suggestion, I will implement this feature 😀.